src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
+    this.name = name;
+    this.health = health;
+    this.hidden = hidden;
```

Comment:
The 'hidden' property should not be initialized in the Animal class constructor, as not all animals will use this property (e.g., Carnivores don't need the 'hidden' property). It's better to initialize 'hidden' directly in the Herbivore class where it's relevant.